src/commands/read.ts

```diff
@@ -45,10 +45,11 @@ export const handler = async (argv: Arguments<Options>): Promise<void> => {
     chain,
   };
   try {
-    const res = await connect(options).read(query);
-    const formatted = format === "raw" ? res : resultsToObjects(res);
+    const res = await connect(options).read(query); // Defaults to "table"
+    // After we upgrade the SDK to version 4.x, we can drop some of this formatting code
+    const formatted = format === "table" ? res : resultsToObjects(res);
 
-    if (format.startsWith("tab")) {
+    if (format === "pretty") {
```

Comment:
Now tabular is renamed to "pretty"... naming things is hard!

Reply:
Love it!